packages/myst-cli/src/transforms/outputs.ts

```diff
@@ -116,7 +128,7 @@ export function transformFilterOutputStreams(
               },
             );
           }
-          return !doRemove;
+          return doRemove;
```

Comment:
Is this intentional?

Reply:
Yes, instead of filtering the set that we keep (and assigning to `children`), it filters the items that we remove, and sets their type to `__delete__`

Reply:
Cool.